doc/build-root.mdoc

```diff
@@ -31,9 +31,9 @@ the concept that allows a developer to keep the source tree clean from
 build products by asking the build system to place such build products
 under a separate subtree.
 .Pp
-Most build systems today support build directories.  For example, the GNU
-Automake/Autoconf build system exposes such concept when invoked as
-follows:
+Most build systems today support build directories.
+For example, the GNU Automake/Autoconf build system exposes such concept when
```

Comment:
Note that the existing text used two spaces after a period which should have done the right thing when rendered (i.e. this should have acted the same way as a newline).

I understand linting tools suggest newline at end of sentence, so OK.

Reply:
For whatever reason, line breaks are the defacto linting rule mandoc 🤷‍♀️.